recipe/meta.yaml

```diff
@@ -33,12 +38,13 @@ requirements:
     - openblas
 
 test:
+  source_files:
+    - runtests.py  
```

Comment:
isn't this enough? @mariusvniekerk

Reply:
That file isn't in the sdist we pull from https://github.com/numpy/numpy/releases/tag/v1.13.3

We can try to get the actual source bundle instead

Reply:
Ah i swear I checked if it was pypy or github.